types/castjs/castjs-tests.ts

```diff
@@ -0,0 +1,71 @@
+import { Castjs } from 'castjs';
```

Comment:
imo it looks incorrect, class symbol is exported directly:
https://github.com/castjs/castjs/blob/master/cast.js#L537

so this is more like kind-of UMD module (works in CJS and in global script)
```
xport = Castjs;
export as namespace Castjs;
```

Reply:
Updated export.

Reply:
I'm getting the error:

```
error  Instead of `export =`-ing a namespace, use the body of the namespace as the module body  export-just-namespace
```

Do I have the code structured as you wanted now? What can I do to bypass this error?

Reply:
try this:
```diff
diff --git a/types/castjs/index.d.ts b/types/castjs/index.d.ts
index caa8db3374..e6c89224b0 100644
--- a/types/castjs/index.d.ts
+++ b/types/castjs/index.d.ts
@@ -3,10 +3,263 @@
 // Definitions by: Adam Stephenson <https://github.com/adstep>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped

-export = castjs;
-export as namespace castjs;
+export = Castjs;
+export as namespace Castjs;
+
+declare class Castjs {
+    /**
+     * Gets the castjs Version
...
```
```diff
+}

-declare namespace castjs {
+declare namespace Castjs {
     interface Options {
         receiver: string;
         joinpolicy: string;
...
```
https://www.typescriptlang.org/docs/handbook/modules.html#zipcodevalidatorts-2

hth